package.json

```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "start": "grunt server --node-env=dev",
     "test": "grunt test --node-env=test",
-    "outdated": "npm outdated --depth 0",
-    "postinstall": "node scripts/gen_keys"
```

Comment:
Hrm, I was trying to avoid shipping private keys in a docker container, but maybe there's another way I can avoid this...